src/plugins/discover/public/plugin.tsx

```diff
@@ -314,6 +323,25 @@ export class DiscoverPlugin
       DiscoverContainer: (props: DiscoverContainerProps) => (
         <DiscoverContainerInternal getDiscoverServices={getDiscoverServicesInternal} {...props} />
       ),
+      logColumns: {
```

Comment:
ℹ️ Please see PR description.

Reply:
Have you considered moving `LazySummaryColumn` and `LogLevelCell` to `@kbn/discover-utils` package?

If the current approach works, it seems like it does not require much of Discover services being hooked up. With `<KibanaContextProvider services={{ core, plugins }}>` it would not be able to access them as `plugins` does not fit `DiscoverServices` structure.

Reply:
This is now done in https://github.com/elastic/kibana/pull/194867/commits/c290819c1c1e1cb5a67d437cca7783c0e2302c8f. 

I did have to introduce a new package to avoid a bunch of circular dependencies.